### ISSUE TITLE
Bug (in Ajenti) - webUI fails to start

### ISSUE THREAD

**Issue by zefanja** [linuxmuster/linuxmuster-webui7#36]
After the initial setup and a reboot, the webUI does not run anymore. The error seems to be related to ajenti. Here is the log:

```
$ cat /root/ajenti-crash.txt
Ajenti bug report
--------------------


Info | Value
----- | -----
Ajenti | 2.1.24
Platform | debian / ubuntu / Ubuntu 18.04.1 LTS
Architecture | x86_64 x86_64
Python | 2.7.15rc1
Debug | False
Loaded plugins | ace, augeas, auth_users, core, dashboard, filesystem, lm_auth, lm_common, lm_devices, lm_linbo, lm_quotas, lm_settings, lm_setup_wizard, lm_users, lm_w_datetime, lmn_groupmembership, lmn_session, network, passwd, plugins, settings

Library | Version
------- | -------
gevent | 1.2.2
greenlet | 0.4.15
psutil | 5.4.7


    Traceback (most recent call last):
      File "/usr/local/lib/python2.7/dist-packages/aj/entry.py", line 30, in start
        aj.core.run(**kwargs)
      File "/usr/local/lib/python2.7/dist-packages/aj/core.py", line 161, in run
        open(aj.config.data['ssl']['certificate']).read()
      File "/usr/local/lib/python2.7/dist-packages/OpenSSL/crypto.py", line 1825, in load_certificate
        _raise_current_error()
      File "/usr/local/lib/python2.7/dist-packages/OpenSSL/_util.py", line 54, in exception_from_error_queue
        raise exception_type(errors)
    Error: [('PEM routines', 'PEM_read_bio', 'no start line')]

```
This is the info from the ajenti log file:

```
2018-10-08 12:02:57,640 INFO    : Loading config from /etc/ajenti/config.yml
2018-10-08 12:02:57,645 INFO    : Ajenti Core 2.1.24
2018-10-08 12:02:57,645 INFO    : Detected platform: debian / Ubuntu 18.04.1 LTS
2018-10-08 12:02:57,715 INFO    : Discovered 22 plugins
2018-10-08 12:02:57,715 WARNING : Not loading [lm_printers] because [lm_workstations] is unavailable
2018-10-08 12:02:57,854 INFO    : Loaded 21 plugins
2018-10-08 12:02:57,855 INFO    : Binding to [0.0.0.0]:8000
2018-10-08 12:02:57,861 ERROR   : Fatal crash occured
2018-10-08 12:02:57,871 INFO    : New worker "restricted session" PID 1453, EUID 0, EGID 0
2018-10-08 12:02:57,872 INFO    : Worker 1453 is demoting to UID 65534 / GID 65534...
2018-10-08 12:02:57,873 INFO    : ...done, new EUID 65534 EGID 65534
2018-10-08 12:02:57,876 ERROR   : Crash report written to /root/ajenti-crash.txt
```

**Comment by zefanja** [comment]
More infos: Looks like the cert linked in config.yml is the problem. On my system it is set to `/etc/linuxmuster/ssl/server.cert.bundle.pem`, but this file is empty. May hostname is different to `server`. Who creates this file? WebUI7 or linuxmuster-setup?

**Comment by zefanja** [comment]
More infos: Looks like the cert linked in config.yml is the problem. On my system it is set to `/etc/linuxmuster/ssl/server.cert.bundle.pem`, but this file is empty. May hostname is different to `server`. Who creates this file? WebUI7 or linuxmuster-setup?

**Comment by netzint** [comment]
The Bundle file is created by webui with:

cat /etc/linuxmuster/ssl/server.key.pem /etc/linuxmuster/ssl/server.cert.pem >  /etc/linuxmuster/ssl/server.cert.bundle.pem

This is likely a bug and have to be fixed with different hostnames.

**Comment by zefanja** [comment]
OK

**Comment by PLanB2008** [comment]
Should be fixed in  ce361bc12b6c5be79014fde0ede930f9ed50c0b4

@zefanja , could you give it a try? I'll create a deb package from the current commit for you to test.

EDIT: just uploaded linuxmuster-webui7_1.0.18-41_all

**Comment by zefanja** [comment]
Fixed, thanks.
